src/CompletionProvider.php

```diff
@@ -237,7 +237,7 @@ public function provideCompletion(PhpDocument $doc, Position $pos): CompletionLi
             );
 
             // Add parent classes
-            $prefixes = $this->expandParentFqns($prefixes);
+            $prefixes = iterator_to_array($this->expandParentFqns($prefixes), false);
```

Comment:
It is not used directly as ```$prefixes``` can be an array, as for removing the ```iterator_to_array``` and collecting the modified keys in an array - will do ASAP.